opta/README.md

```diff
@@ -0,0 +1,7 @@
+# Easy Flyte Deployment with Opta
```

Comment:
@SandraGH5, do you mind proof-reading this file and suggesting edits as you see fit?

Reply:
Done.